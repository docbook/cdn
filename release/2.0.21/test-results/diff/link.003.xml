<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#section">1. Section Title</a>
                     </span>
                  </li>
                  <li>
                     <span>
                        <a href="#section2">2. Long Section Title</a>
                     </span>
                  </li>
               </ul>
            </div>
         </div>
         <section deltaxml:deltaV2="A" id="section" class="section">
            <div class="section-titlepage">
               <h2>1. Section Title</h2>
            </div>
            <p>A <a href="#section">self-referential link</a>.</p>
            <p>A link without content: <a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p>
            <p>A link without content, but with an xreflabel:
<a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p>
            <p>A link with content and an xreflabel:
<a href="#section2">here</a>.</p>
         </section><div deltaxml:deltaV2="B" class="content">
            <section id="section" class="section">
               <div class="section-titlepage">
                  <h2>1. Section Title</h2>
               </div>
               <div class="content">
                  <p>A <a href="#section">self-referential link</a>.</p>
                  <p>A link without content: <a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p>
                  <p>A link without content, but with an xreflabel:
<a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p>
                  <p>A link with content and an xreflabel:
<a href="#section2">here</a>.</p>
               </div>
            </section>
            <section id="section2" class="section">
               <div class="section-titlepage">
                  <h2>2. A Very Long Section Title That I Don't Want To Use For My XRef</h2>
               </div>
               <div class="content">
                  <p>Some text with a <a href="#section">link back</a> to the
first section.</p>
               </div>
            </section>
         </div><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="section2" class="section">
            <div class="section-titlepage">
               <h2>2. A Very Long Section Title That I Don't Want To Use For My XRef</h2>
            </div>
            <p>Some text with a <a href="#section">link back</a> to the
first section.</p>
         </section>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>