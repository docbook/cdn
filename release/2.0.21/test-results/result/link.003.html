<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for link.003.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: link.003.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">3</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#section"</span>&gt;1. Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#section2"</span>&gt;2. Long Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff del">&lt;section <span class="att">class="section"</span> <span class="att">id="section"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;1. Section Title&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;A <span class="tag">&lt;a <span class="att">href="#section"</span>&gt;self-referential link&lt;/a&gt;</span>.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;A link without content: <span class="tag">&lt;a <span class="att">href="#section2"</span> <span class="att">title="A Very Long Section Title That I Don't Want To Use For My XRef"</span>&gt;Section 2, “Long Section Title”&lt;/a&gt;</span>.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;A link without content, but with an xreflabel:
<span class="tag">&lt;a <span class="att">href="#section2"</span> <span class="att">title="A Very Long Section Title That I Don't Want To Use For My XRef"</span>&gt;Section 2, “Long Section Title”&lt;/a&gt;</span>.&lt;/p&gt;</span>
            <span class="tag">&lt;p&gt;A link with content and an xreflabel:
<span class="tag">&lt;a <span class="att">href="#section2"</span>&gt;here&lt;/a&gt;</span>.&lt;/p&gt;</span>
         &lt;/section&gt;</span><span class="diff txtchg">
         </span><span class="tag diff del">&lt;section <span class="att">class="section"</span> <span class="att">id="section2"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
               <span class="tag">&lt;h2&gt;2. A Very Long Section Title That I Don't Want To Use For My XRef&lt;/h2&gt;</span>
            &lt;/div&gt;</span>
            <span class="tag">&lt;p&gt;Some text with a <span class="tag">&lt;a <span class="att">href="#section"</span>&gt;link back&lt;/a&gt;</span> to the
first section.&lt;/p&gt;</span>
         &lt;/section&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag nodiff">&lt;div <span class="att">class="lists-of-titles"</span>&gt;
            <span class="tag">&lt;div <span class="att">class="toc"</span>&gt;
               <span class="tag">&lt;p&gt;
                  <span class="tag">&lt;b&gt;Table of Contents&lt;/b&gt;</span>
               &lt;/p&gt;</span>
               <span class="tag">&lt;ul <span class="att">class="toc"</span>&gt;
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#section"</span>&gt;1. Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
                  <span class="tag">&lt;li&gt;
                     <span class="tag">&lt;span&gt;
                        <span class="tag">&lt;a <span class="att">href="#section2"</span>&gt;2. Long Section Title&lt;/a&gt;</span>
                     &lt;/span&gt;</span>
                  &lt;/li&gt;</span>
               &lt;/ul&gt;</span>
            &lt;/div&gt;</span>
         &lt;/div&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;section <span class="att">class="section"</span> <span class="att">id="section"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;1. Section Title&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;A <span class="tag">&lt;a <span class="att">href="#section"</span>&gt;self-referential link&lt;/a&gt;</span>.&lt;/p&gt;</span>
                  <span class="tag">&lt;p&gt;A link without content: <span class="tag">&lt;a <span class="att">href="#section2"</span> <span class="att">title="A Very Long Section Title That I Don't Want To Use For My XRef"</span>&gt;Section 2, “Long Section Title”&lt;/a&gt;</span>.&lt;/p&gt;</span>
                  <span class="tag">&lt;p&gt;A link without content, but with an xreflabel:
<span class="tag">&lt;a <span class="att">href="#section2"</span> <span class="att">title="A Very Long Section Title That I Don't Want To Use For My XRef"</span>&gt;Section 2, “Long Section Title”&lt;/a&gt;</span>.&lt;/p&gt;</span>
                  <span class="tag">&lt;p&gt;A link with content and an xreflabel:
<span class="tag">&lt;a <span class="att">href="#section2"</span>&gt;here&lt;/a&gt;</span>.&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/section&gt;</span>
            <span class="tag">&lt;section <span class="att">class="section"</span> <span class="att">id="section2"</span>&gt;
               <span class="tag">&lt;div <span class="att">class="section-titlepage"</span>&gt;
                  <span class="tag">&lt;h2&gt;2. A Very Long Section Title That I Don't Want To Use For My XRef&lt;/h2&gt;</span>
               &lt;/div&gt;</span>
               <span class="tag">&lt;div <span class="att">class="content"</span>&gt;
                  <span class="tag">&lt;p&gt;Some text with a <span class="tag">&lt;a <span class="att">href="#section"</span>&gt;link back&lt;/a&gt;</span> to the
first section.&lt;/p&gt;</span>
               &lt;/div&gt;</span>
            &lt;/section&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#section">1. Section Title</a></span></li><li><span><a href="#section2">2. Long Section Title</a></span></li></ul></div></div><section id="section" class="section"><div class="section-titlepage"><h2>1. Section Title</h2></div><p>A <a href="#section">self-referential link</a>.</p><p>A link without content: <a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p><p>A link without content, but with an xreflabel:
<a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p><p>A link with content and an xreflabel:
<a href="#section2">here</a>.</p></section><section id="section2" class="section"><div class="section-titlepage"><h2>2. A Very Long Section Title That I Don't Want To Use For My XRef</h2></div><p>Some text with a <a href="#section">link back</a> to the
first section.</p></section></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#section">1. Section Title</a></span></li><li><span><a href="#section2">2. Long Section Title</a></span></li></ul></div></div><div class="content"><section id="section" class="section"><div class="section-titlepage"><h2>1. Section Title</h2></div><div class="content"><p>A <a href="#section">self-referential link</a>.</p><p>A link without content: <a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p><p>A link without content, but with an xreflabel:
<a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p><p>A link with content and an xreflabel:
<a href="#section2">here</a>.</p></div></section><section id="section2" class="section"><div class="section-titlepage"><h2>2. A Very Long Section Title That I Don't Want To Use For My XRef</h2></div><div class="content"><p>Some text with a <a href="#section">link back</a> to the
first section.</p></div></section></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/link.003.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>