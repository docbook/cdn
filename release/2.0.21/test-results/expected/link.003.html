<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#section">1. Section Title</a></span></li><li><span><a href="#section2">2. Long Section Title</a></span></li></ul></div></div><section id="section" class="section"><div class="section-titlepage"><h2>1. Section Title</h2></div><p>A <a href="#section">self-referential link</a>.</p><p>A link without content: <a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p><p>A link without content, but with an xreflabel:
<a href="#section2" title="A Very Long Section Title That I Don't Want To Use For My XRef">Section 2, “Long Section Title”</a>.</p><p>A link with content and an xreflabel:
<a href="#section2">here</a>.</p></section><section id="section2" class="section"><div class="section-titlepage"><h2>2. A Very Long Section Title That I Don't Want To Use For My XRef</h2></div><p>Some text with a <a href="#section">link back</a> to the
first section.</p></section></article><script src="../../resources/js/prism.js"></script></body></html>